haproxy/templates/default/haproxy.cfg.frontend.erb

```diff
@@ -1,33 +1,36 @@
-
+<%
+haproxy_cfg = @node.fetch('haproxy', {})
+opsworks_layers = @node.fetch('opsworks', {}).fetch('layers', {})
+%>
 frontend http-in
   bind *:80
-  <% unless @node.fetch('haproxy',{}).fetch('ssl', 'off') == 'off'
+  <% unless haproxy_cfg.fetch('ssl', 'off') == 'off'
     certificate = "#{@node['ssl-deploy']['directory']}/cert.combined.pem"
   -%>
   bind *:443 ssl crt <%= certificate %>
   reqadd X-Forwarded-Proto:\ https if { ssl_fc }
-  <% unless @node.fetch('haproxy',{}).fetch('hsts',{}).nil? -%>
-  rspadd Strict-Transport-Security:\ max-age=<%=@node.fetch('haproxy',{}).fetch('hsts',{})%> if { ssl_fc }
+  <% unless haproxy_cfg.fetch('hsts',{}).nil? -%>
+  rspadd Strict-Transport-Security:\ max-age=<%=haproxy_cfg.fetch('hsts',{})%> if { ssl_fc }
   <% end
-     if @node.fetch('haproxy',{}).fetch('ssl',{}) == 'only' -%>
+     if haproxy_cfg.fetch('ssl',{}) == 'only' -%>
```

Comment:
@fh Sort of the same here. Default is an object. Maybe it should default to whatever "ssl" could be? Or what the default is for that.